org.idempiere.test/src/org/idempiere/test/adwindow/GridTabTest.java

```diff
@@ -253,6 +252,8 @@ public void dataStatusChanged(DataStatusEvent e) {
 			assertTrue(gTab0.dataNew(false));
 			assertTrue(gTab0.isNew(), "Grid Tab dataNew call not working as expected");
 			gTab0.setValue(MOrder.COLUMNNAME_C_BPartner_ID, DictionaryIDs.C_BPartner.C_AND_W.id);
+			gTab0.setValue(MOrder.COLUMNNAME_C_DocTypeTarget_ID, DictionaryIDs.C_DocType.STANDARD_ORDER.id);
```

Comment:
These two setValue are harmless - but are not required as they are initialized with the user preferences

Reply:
I didn't investigate but in my local testing, the loading of user preference doesn't works for me. Anyway, I think it is better not to rely on that in unit testing..